src/shared/components/ui/Divider/Divider.js

```diff
@@ -1,41 +1,39 @@
-import { styles } from '@helpers/css'
 import PropTypes from 'prop-types'
 import React from 'react'
-import scss from './Divider.scss'
-
-const cssClass = styles(scss)
+import { useStyles } from '@helpers/hooks/useStyles'
+import styles from './Divider.module.scss'
 
 /**
  * Divider - stateless presentational component
  * @param {object} props - props
- * @param {string|array} props.className - a list of class names, default: `divider`
  * @param {string} props.variant - variant `normal, menu, dropdown, horizontal`
  * @param {string} props.align - align `vertical`
  * @return {object} An object of children element
  */
-const Divider = ({ className, variant, align }) => {
-  const elementClasses = cssClass({
-    'divider--dropdown': variant === 'dropdown',
-    'divider--horizontal': variant === 'horizontal',
-    'divider--menu': variant === 'menu',
-    'divider--normal': variant === 'normal',
-    'divider--vertical': align === 'vertical'
+const Divider = ({ variant, align }) => {
+  const elementClasses = useStyles({
+    [styles['divider']]: true,
+    [styles['divider--dropdown']]: variant === 'dropdown',
```

Comment:
a może po prostu divider + variant :p

Reply:
Problem z tym, że mi to nie dodaje kurcze tych klas...ale sprawdzę to na spokojnie w poniedziałek bo już mózg wysiadł od tych ui-kitów